atorch/examples/llama2/README.md

```diff
@@ -337,13 +387,16 @@
       - `ds_config`: `Dict` for deepspeed config or `str` for json file path. 
       - `batch_fn`:  input adaptation function for deepspeed pipeline (inputs,), (labels,). 
 
-## Example
-llama2
-- file entrance
-[ds_3d_llama2.py](ds_3d_llama2.py)
+### Scripts
+
+- training file [ds_3d_llama2.py](ds_3d_llama2.py)
+
+- launch script [ds_3d_llama2_entry.sh](ds_3d_llama2_entry.sh)
 
-- starup script
-[llama2_entry.sh](llama2_entry.sh)
+```bash
+cd dlrover/atorch/examples/Llama2
+pip install -r requirements.txt
 
-- utils
-[utils.py](utils.py)
\ No newline at end of file
+# Configurable environment variable: DATASET_PATH, MODEL_NAME_OR_PATH, PIPELINE_PARALLEL_SIZE, MODEL_PARALLEL_SIZE, etc.
```

Comment:
Explain how to set 3D parallel dimensions. Give an example of how to run in 8-gpu system.

Also make the example runnable. Currently, it crashes with 2x2x2 3D.